publish.gradle

```diff
@@ -200,6 +233,14 @@ model {
                 groupId artifactGroupId
                 version pubVersion
             }
+
+            vendordep(MavenPublication) {
+                artifact vendordepJsonZip
+
+                artifactId = "${baseArtifactId}-vendordep"
```

Comment:
You can actually publish the vendordep json directly. No need to zip it.

Reply:
How would i? I had tried and it would constantly fail.

Reply:
Do i even really need to. I only added this in response to @rzblue I dont really think its necessary

Reply:
I guess the zip is fine. So I'll add some other comments on things to fix.